tests/multinode.rs

```diff
@@ -789,6 +823,11 @@ fn test_multi_node_dynamic_network() {
         }
 
         while duration_as_ms(&now.elapsed()) < confirmation as u64 {
+            info!(
+                "while duration_as_ms: {} < {}",
+                duration_as_ms(&now.elapsed()),
+                confirmation
+            );
             sleep(Duration::from_millis(100));
             confirmation = client.get_confirmation_time()
```

Comment:
Any way you can boot all calls to this method while you're in the neighborhood?  It doesn't return a very meaningful number and this code probably predates `confirm_transaction()`.

Reply:
Interestingly this part of the test is still busted.  Because none of the nodes in this test actually stake, `get_confirmation_time()` comes back as `std::usize::MAX` and the test never exits 💀